package.json

```diff
@@ -252,6 +257,11 @@
                 "key": "ctrl+Enter",
                 "when": "editorTextFocus && editorLangId == julia"
             },
+            {
+                "command": "language-julia.executeJuliaCellInREPL",
+                "key": "shift+Enter",
```

Comment:
I think one worry here is that this is already a valid keybinding in the VS Code editor...

Reply:
One other idea: we could re-use Alt+Enter for this? If you are in a cell, it executes a cell, if not, it executes a code block? That would also make it compatible with Juno, at least partially?

Reply:
`Shift+Enter` has no conflicts in the Julia context. There are analogous bindings for python in the `editorTextFocus` scope, but they don't conflict because of the distinct `editorLangId`. The only other binding is in the `inSettingsSearch` scope.

In contrast, `Alt+Enter` _would_ conflict with block execution. There is always a cell that can be uniquely associated with the cursor position.

Reply:
That said, I'd prefer if the bindings could be as consistent as possible with Juno.

Reply:
Ah, you are right, somehow I thought `Shift+Enter` did something different in the editor, so that isn't a problem.

> In contrast, `Alt+Enter` would conflict with block execution. There is always a cell that can be uniquely associated with the cursor position.

I guess this would amount to saying that if a file has even just a single `##` in it, then `Alt+Enter` executes cells, otherwise it executes code blocks. That doesn't seem like a great choice, though.

So maybe we just stick with the choice you made here for now, and then at some point in the distant future we can re-evaluate all the code execution keybindings.